packages/react-shortcuts/src/ShortcutProvider/ShortcutProvider.tsx

```diff
@@ -1,34 +1,30 @@
 import * as React from 'react';
-import PropTypes from 'prop-types';
 import ShortcutManager from '../ShortcutManager';
 
-export const contextTypes = {
-  shortcutManager: PropTypes.instanceOf(ShortcutManager),
-};
-
 export interface Context {
-  shortcutManager: ShortcutManager;
+  shortcutManager?: ShortcutManager;
 }
 
 export interface Props {
   children?: React.ReactNode;
 }
 
+export const {Provider, Consumer} = React.createContext<Context>({});
+
 export default class ShortcutProvider extends React.Component<Props, never> {
-  static childContextTypes = contextTypes;
   private shortcutManager = new ShortcutManager();
 
   componentDidMount() {
     this.shortcutManager.setup();
   }
 
-  getChildContext() {
+  get childContext() {
```

Comment:
Should this be private?

Reply:
I think this can actually be removed completely and have context created inside the render function.